docs/blog/2017-03-08-gatsbygram-case-study/index.md

```diff
@@ -317,14 +258,19 @@ export const pageQuery = `
 `
 ```
 
-### React.js pages
+## Creating one-off pages
 
-Gatsby lets you build pages from individual React.js components. Like
-template components, you can add GraphQL queries to query for data.
+In addition to creating a page for every one of our instagram images, we also want to make an index page that shows off our insta grid. To do so, Gatsby lets us define pages using the file system, just like your standard static site generators:
+```
+pages/
+  index.js
+  about.js
+```
+
+These react components can query our graphql schema for data, and will each be generated into their own pages at `gatsbygram.com/` and `gatsbygram.com/about`.
```

Comment:
"our graphql schema" a bit awkward — perhaps "query the Gatsbygram GraphQL schema" or just "query their site's GraphQL schema".